composables/api/useGetLeaderboardBySlug/index.ts

```diff
@@ -14,23 +14,22 @@ export default async function useGetLeaderboardBySlug(
 ): Promise<ApiResponse<LeaderboardViewModel>> {
   const { onError, onOkay } = opts
   const responseData = ref<LeaderboardViewModel>({
-    categories: [],
     id: -1,
     name: '',
     slug: '',
     info: '',
     createdAt: '',
     updatedAt: null,
     deletedAt: null,
+    status: 'Published',
```

Comment:
The viewmodel has this new field which currently has "Published" and "Deleted" to denote not-deleted and deleted boards, respectively.